### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by radex**

### CODE REVIEW

**Comment by radex** [comment]
@rozPierog fucking travis broke on Android again 😭

File: .travis.yml [hunk 1]
```diff
@@ -35,10 +35,8 @@ matrix:
     - language: android
       android:
         components:
-          - build-tools-28.0.3
-          - android-28
-          - android-24
```
Comment (rozPierog) [inline .travis.yml]: No wonders it broke. You removed emulator image from components 🙊

Comment (radex) [reply .travis.yml]: nah man it's broken on master

Comment (rozPierog) [reply .travis.yml]: Yeah, I figured it out later. Tried to fix this (look at commits below) but sadly nothing works. May look at this at a later date (like tomorrow)

Comment (radex) [reply .travis.yml]: thanks for trying ❤️ 

fucking travis.

maybe it's time to look at other RN projects, see how they handle CI? And maybe ditch Travis in favor of Circle?

**Pull request closed without merging**